webshop-backend/src/Validator.java

```diff
@@ -0,0 +1,29 @@
+public class Validator {
+
+    public static boolean sysIsNameValid(String name) {
+        return isNameValid(name);
+    }
+
+    public static boolean sysPriceValid(int price) {
+        return isPriceValid(price);
+    }
+
+    public static boolean sysStockValid(int stock) {
+        return isStockValid(stock);
+    }
+
+    private static boolean isNameValid(String name) {
+        return name.matches("[a-z ]+$");
+
+    }
+
+    private static boolean isPriceValid(int price) {
+        return price > 0;
+    }
+
+    private static boolean isStockValid(int stock) {
+        return stock >= 0;
+    }
+
```

Comment:
Good , but why did you do it like this?
I don't get it. You could've just validated those things in your first 3 methods right away. I don't see the need of those private methods.